scripts/get_das_info.py

```diff
@@ -207,6 +224,58 @@ def get_das_info(
             info_of_interest["nfiles"] = dataset_info.get("nfiles", "")
             info_of_interest["nevents"] = dataset_info.get("nevents", "")
 
+    if add_file_info:
+        file_infos = load_das_info(dataset, add_file_info=True)
+
+        empty_files = [
+            info["file"][0]["name"]
+            for info in filter(lambda info: info["file"][0]["nevents"] == 0, file_infos)
+        ]
+        broken_files = [
+            info["file"][0]["name"]
+            for info in filter(lambda info: info["file"][0]["is_file_valid"] == 0, file_infos)
+        ]
+        info_of_interest["empty_files"] = empty_files
+        info_of_interest["broken_files"] = broken_files
+    else:
+        info_of_interest["empty_files"] = "UNKNOWN"
+        info_of_interest["broken_files"] = "UNKNOWN"
+
+    return info_of_interest
+
+
+def new_get_das_info(dataset: str) -> dict:
+    info_of_interest = {"name": dataset}
+
+    file_infos = load_das_info(dataset, add_file_info=True)
+
+    info_of_interest["dataset_id"] = file_infos[0]["file"][0]["dataset_id"]
+
+    empty_files_filter = lambda info: info["file"][0]["nevents"] == 0
+    broken_files_filter = lambda info: info["file"][0]["is_file_valid"] == 0
+
+    good_files = list(filter(lambda x: not broken_files_filter(x) and not empty_files_filter(x), file_infos))
+
+    dataset_id = {info["file"][0]["dataset_id"] for info in good_files}
+    if len(dataset_id) == 1:
+        info_of_interest["dataset_id"] = dataset_id.pop()
+    else:
+        raise ValueError(f"Multiple dataset IDs ({dataset_id}) found for dataset {dataset}")
+
+    info_of_interest["nfiles"] = len(good_files)
+    info_of_interest["nevents"] = sum(info["file"][0]["nevents"] for info in good_files)
+
+    empty_files = [
+        info["file"][0]["name"]
+        for info in filter(empty_files_filter, file_infos)
+    ]
+    broken_files = [
+        info["file"][0]["name"]
+        for info in filter(broken_files_filter, file_infos)
+    ]
+    info_of_interest["empty_files"] = empty_files
+    info_of_interest["broken_files"] = broken_files
```

Comment:
isn't this already part of the `load_das_info` function? Why do it again here?

Reply:
there are currently two function doing basically the same to allow runtime comparisons (`get_das_info` and `new_get_das_info`). One of them should be removed before merging this PR